### ISSUE TITLE
Issue with Rails 5 and Sprockets 4

### ISSUE THREAD

**Issue by olliekav** [railsware/js-routes#203]
I've been trying to upgrade my app to Rails 5 but Activeadmin requires a Sprockets update to 4 with sass-rails. As a result I'm getting this error with JS-Routes...

`undefined method 'call' for :"js-routes_dependent_on_routes":Symbol`

Any ideas why this might be?

**Comment by Schwad** [comment]
I have gotten this exact same issue, thank you. 

I have stackoverflowed it here: http://stackoverflow.com/questions/38747650/rails-5-upgrade-nomethoderror-undefined-method-call-for-js-routes-dependen

No answers yet. 

Wondering if it has something to do with here?

https://github.com/railsware/js-routes/blob/master/lib/js_routes/engine.rb

**Comment by Schwad** [comment]
I'm wondering if this has something to do with the config/manifest.js file and needing to move our js-routes references. Not sure, still broken.

**Comment by olliekav** [comment]
@Schwad You just solved the problem with the link to that `engine.rb` file. I just forked it to test and updated the sprockets version and I'm up and running...

https://github.com/WaterRangers/js-routes/blob/master/lib/js_routes/engine.rb

Guess we can submit a PR for this.

**Comment by Schwad** [comment]
Awesome :)

**Comment by Schwad** [comment]
I'm going to point to your fork to test on my end.

**Comment by Schwad** [comment]
So the good news:

1. With your change js-routes no longer seems to be breaking my applications

Bad news:

2. It's still broken, on to hunt this bug down elsewhere 🎱 

```ruby
ExecJS::RuntimeError at /users/sign_in
SyntaxError: [stdin]:15:19: missing / (unclosed regex)
```

**Comment by bogdan** [comment]
@Schwad please close this one than and open a new one with your new problem

**Comment by Schwad** [comment]
edited it out. I don't have auth to close this issue.

**Comment by Schwad** [comment]
I'm happy for this issue to be closed when @olliekav puts together a pull request.

**Comment by flood4life** [comment]
I think #202 solves this issue too.

**Comment by Schwad** [comment]
Okay, I'm incorporating #202's proposed changes on my fork, will have a look. https://github.com/schwad/js-routes

**Comment by Schwad** [comment]
Currently pointing to those changes and not seeing js-routes related breaks.... for now...

**Comment by bogdan** [comment]
New version released with #202 merged.

**Comment by olliekav** [comment]
@Schwad @bogdan Perfect, working all fine for me. Closing this.
